pipeline_api/src/main/java/com/splicemachine/pipeline/writehandler/UpdateUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.splicemachine.pipeline.writehandler;
+
+import com.carrotsearch.hppc.BitSet;
+import com.carrotsearch.hppc.BitSetIterator;
+import com.splicemachine.encoding.MultiFieldDecoder;
+import com.splicemachine.kvpair.KVPair;
+import com.splicemachine.storage.EntryDecoder;
+import com.splicemachine.storage.index.BitIndex;
+import com.splicemachine.storage.index.BitIndexing;
+import com.splicemachine.utils.ByteSlice;
+
+public class UpdateUtils {
+
+    public static KVPair deleteFromUpdate(KVPair mutation) {
```

Comment:
this looks like it can be easily unit-tested. please add a unit test and a comment what this should do.

Reply:
Added explanation and unit test